diff -urN gucharmap-1.3.1.orig/configure.ac gucharmap-1.3.1/configure.ac
--- gucharmap-1.3.1.orig/configure.ac	2004-03-04 09:21:24.691516064 +0100
+++ gucharmap-1.3.1/configure.ac	2004-03-04 09:30:46.289140248 +0100
@@ -36,7 +36,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", 
                    [Define the gettext package to be used])
-ALL_LINGUAS="am ar az ca be bn cs cy da de el en_CA en_GB eo es fa fi fr ga hi hr hu id is it ja ko lt mk ml mn ms nl nn no pa pl pt pt_BR ru sk sq sr sr@Latn sv ta th uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar az ca be bn cs cy da de el en_CA en_GB eo es fa fi fr ga hi hr hu id is it ja ko lt mk ml mn ms nb nl nn pa pl pt pt_BR ru sk sq sr sr@Latn sv ta th uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 
