diff -urN gucharmap-0.8.0.orig/gucharmap/Makefile.am gucharmap-0.8.0/gucharmap/Makefile.am
--- gucharmap-0.8.0.orig/gucharmap/Makefile.am	Wed Jun 11 07:31:09 2003
+++ gucharmap-0.8.0/gucharmap/Makefile.am	Wed Jun 11 07:33:03 2003
@@ -132,7 +132,7 @@
 	$(GTK_QUERY_IMMODULES) $(module_LTLIBRARIES) > gtk.immodules ;
 
 install-data-hook: im-module-file
-	LD_LIBRARY_PATH=$(DESTDIR)$(libdir):$$LD_LIBRARY_PATH GTK_PATH=$$GTK_PATH:$(DESTDIR)$(libdir)/gtk-2.0 $(GTK_QUERY_IMMODULES) > `./im-module-file` ;
+	LD_LIBRARY_PATH=$(DESTDIR)$(libdir):$$LD_LIBRARY_PATH GTK_PATH=$$GTK_PATH:$(DESTDIR)$(libdir)/gtk-2.0 $(GTK_QUERY_IMMODULES) > $(DESTDIR)`./im-module-file` ;
 
 else
 
