diff -urN gucharmap-1.4.2.orig/gucharmap.desktop.in gucharmap-1.4.2/gucharmap.desktop.in
--- gucharmap-1.4.2.orig/gucharmap.desktop.in	2004-12-03 15:35:14.674300184 +0100
+++ gucharmap-1.4.2/gucharmap.desktop.in	2004-12-03 15:35:31.984668608 +0100
@@ -6,7 +6,7 @@
 Icon=gucharmap.png
 Terminal=false
 Type=Application
-Categories=Application;Utility;
+Categories=GTK;GNOME;Utility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gucharmap
 X-GNOME-Bugzilla-Component=general
