diff -aurN gucharmap-1.5.1.orig/gucharmap.desktop.in gucharmap-1.5.1/gucharmap.desktop.in
--- gucharmap-1.5.1.orig/gucharmap.desktop.in	2006-01-22 04:06:26.000000000 +0100
+++ gucharmap-1.5.1/gucharmap.desktop.in	2006-01-29 22:31:12.783612000 +0100
@@ -6,7 +6,7 @@
 Icon=gucharmap.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Utility;
+Categories=GTK;GNOME;Utility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gucharmap
 X-GNOME-Bugzilla-Component=general
